app/javascript/spree/dashboard/index.js

```diff
@@ -6,14 +6,12 @@ class Dashboard {
   }
 }
 
+import * as Turbo from "@hotwired/turbo"
```

Comment:
@MatthewKennedy we cannot have turbo import here, new applications have turbo pre-installed and this will cause double turbo initialization and errors